src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,197 @@
+var malawi_binary = ee.Image("projects/ee-aasareansah/assets/malawi_binary"),
+    Sentinel2 = ee.Image("projects/ee-aasareansah/assets/Sent_Sept_Comp_Malawi"),
+    Malawi = ee.FeatureCollection("FAO/GAUL/2015/level0").filter(ee.Filter.eq('ADM0_NAME', 'Malawi')),
+    Malawi_EPA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_EPAs"),
+    Malawi_TA = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_TAs"),
+    Admin = ee.FeatureCollection("projects/ee-aasareansah/assets/NASA_Harvest/Malawi_admin2");
+
+var upd_name = malawi_binary.select('b1').rename('Wrong value');
+
+var leftMap = ui.Map();
+leftMap.centerObject(malawi_binary.geometry().centroid(), 7);
+var layer1 = ui.Map.GeometryLayer([], 'Geometry Layer', 'red');
+var drawingTools1 = leftMap.drawingTools();
+drawingTools1.layers().add(layer1);
+
+var rightMap = ui.Map();
+rightMap.setOptions('satellite');
+
+var layer2 = ui.Map.Layer(ee.FeatureCollection([]), {'color': 'red'}, 'Points');
+
+drawingTools1.onDraw(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.onEdit(ui.util.debounce(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+}, 100));
+drawingTools1.onErase(function(geometry, layer) {
+  if (layer === layer1) {
+    layer2.setEeObject(layer.getEeObject());
+  }
+});
+drawingTools1.setDrawModes(['point']);
+
+var empty = ee.Image().byte();
+var Malawi_color = empty.paint({featureCollection: Malawi, color: 1, width: 2});
+var Admin_color = empty.paint({featureCollection: Admin, color: 1, width: 2});
+var EPA_color = empty.paint({featureCollection: Malawi_EPA, color: 1, width: 2});
+var TA_color = empty.paint({featureCollection: Malawi_TA, color: 1, width: 2});
+
+var palettes = require('users/gena/packages:palettes');
+var palette = palettes.cmocean.Speed[7];
+
+var vis_crop = {min: 0, max: 1.0, palette: palette.slice(0,-2)};
+leftMap.addLayer(Malawi_color, {palette:'cyan'}, 'Malawi border');
+leftMap.addLayer(malawi_binary, vis_crop, 'Malawi cropland');
+leftMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+leftMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_Extension_Planning_Area', false);
+leftMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_Traditional_Authority', false);
+
+var vis = {min: 0.0, max: 0.3, bands: ['B4', 'B3', 'B2']};
+rightMap.addLayer(Sentinel2, vis, 'Sentinel-2');
+rightMap.add(layer2);
+rightMap.addLayer(Malawi_color, {palette:'cyan'}, 'Malawi border');
+rightMap.addLayer(Admin_color, {palette: 'CE33FF'}, 'Malawi_Admin');
+rightMap.addLayer(EPA_color, {palette: '0096FF'}, 'Malawi_Extension_Planning_Area', false);
+rightMap.addLayer(TA_color, {palette: 'black'}, 'Malawi_Traditional_Authority', false);
```

Comment:
This looks a lot better! I think it would require even fewer lines changes if you did something like:
```javascript
var country = "Malawi"
```

```javascript
rightMap.addLayer(Malawi_color, {palette:'cyan'}, country + ' border');
rightMap.addLayer(Admin_color, {palette: 'CE33FF'}, country + '_Admin');
rightMap.addLayer(EPA_color, {palette: '0096FF'}, country + '_Extension_Planning_Area', false);
rightMap.addLayer(TA_color, {palette: 'black'}, country + '_Traditional_Authority', false);
```

Reply:
Cool!